services/requester/requester.go

```diff
@@ -380,6 +387,50 @@ func (e *EVM) EstimateGas(ctx context.Context, data []byte) (uint64, error) {
 	return gasUsed, nil
 }
 
+func (e *EVM) GetCode(
+	ctx context.Context,
+	address common.Address,
+	height uint64,
+) ([]byte, error) {
+	e.logger.Debug().
+		Str("addess", address.Hex()).
+		Str("height", fmt.Sprintf("%d", height)).
+		Msg("get code")
+
+	hexEncodedAddress, err := cadence.NewString(
+		strings.TrimPrefix(address.Hex(), "0x"),
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	value, err := e.client.ExecuteScriptAtLatestBlock(
+		ctx,
+		e.replaceAddresses(getCodeScript),
+		[]cadence.Value{hexEncodedAddress},
+	)
+	if err != nil {
+		return nil, fmt.Errorf("failed to execute script for get code: %w", err)
+	}
+
+	if _, ok := value.(cadence.String); !ok {
+		e.logger.Panic().Msg(fmt.Sprintf("failed to convert value to hex-encoded string: %v", value))
+	}
+
+	hexEncodedCode := value.ToGoValue().(string)
+	code, err := hex.DecodeString(hexEncodedCode)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert string to byte code: %w", err)
+	}
```

Comment:
maybe same for decode, a helper function that is something like: `cadenceStringToBytes()`

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/130/commits/3929b93a2a772ee336bef963ed5f2605da62447f